app/controllers/support_interface/users_controller.rb

```diff
@@ -12,13 +12,59 @@ def index
     def show
       @teacher = IdentityApi.get_teacher(uuid)
       @teacher_name =
-        "#{@teacher.fetch(:firstName)} #{@teacher.fetch(:lastName)}"
+        "#{@teacher.fetch("firstName")} #{@teacher.fetch("lastName")}"
+    rescue IdentityApi::ApiError,
+           Faraday::ConnectionFailed,
+           Faraday::TimeoutError => e
+      Sentry.capture_exception(e)
+      render "errors/internal_server_error", status: :internal_server_error
```

Comment:
This code doesn't do anything; without it, the exact same would happen, as in we'd capture a sentry exception and show the 500 page. I think this pattern was added as part of something else and now is being used in the rest of the codebase, but it's not great...

There should never be a `rescue` followed by `render internal_server_error`